### PR TITLE
Remove authentication step from cart page

### PR DESCRIPTION
**Pull request by alebedev**
## Describe your changes

- Remove returning member authentication before proceeding to checkout
- Implement signing by returning member (get token -> authenticate session -> sign).  Very basic support, UI would be updated in following PRs

## Justify why they are needed

Allow returning members to sign if they skipped login in price calculator

## Checklist before requesting a review

- [ ] I have performed a self-review of my code

### CODE REVIEW

**Comment by alebedev** [comment]
Current dependencies on/for this PR:
* main
  * **PR #1525** <a href="https://app.graphite.dev/github/pr/HedvigInsurance/racoon/1525" target="_blank">Graphite</a> 
    * **PR #1529** <a href="https://app.graphite.dev/github/pr/HedvigInsurance/racoon/1529" target="_blank">Graphite</a> 
      * **PR #1549** <a href="https://app.graphite.dev/github/pr/HedvigInsurance/racoon/1549" target="_blank">Graphite</a> 
        * **PR #1554** <a href="https://app.graphite.dev/github/pr/HedvigInsurance/racoon/1554" target="_blank">Graphite</a>  👈

This comment was auto-generated by [Graphite](https://app.graphite.dev/github/pr/HedvigInsurance/racoon/1554?utm_source=stack-comment).

File: apps/store/src/components/CheckoutHeader/CheckoutHeader.helpers.ts [hunk 1]
```diff
@@ -36,7 +36,7 @@ export const fetchCheckoutSteps = async ({ apolloClient, shopSession }: Params)
   if (showSwitchingAssistant) steps.push(CheckoutStep.SwitchingAssistant)
   if (steps.length < 3) steps.push(CheckoutStep.Confirmation)
   if (steps.length < 3) steps.push(CheckoutStep.Done)
-
+  console.log('steps', steps.length)
```
Comment (robinandeer) [inline apps/store/src/components/CheckoutHeader/CheckoutHeader.helpers.ts:39]: remove debug comment

File: apps/store/src/components/CheckoutPage/useHandleSubmitCheckout.ts [hunk 2]
```diff
@@ -28,23 +34,48 @@ export const useHandleSubmitCheckout = (params: Params) => {
     shopSessionId,
   })
 
+  const [loginState, startLogin] = useBankIdLogin({
+    shopSessionId,
+    ssn,
+    onCompleted() {
+      startSign()
+    },
```
Comment (robinandeer) [inline apps/store/src/components/CheckoutPage/useHandleSubmitCheckout.ts:42]: very nice and declarative 😌

File: apps/store/src/pages/checkout/index.tsx [hunk 3]
```diff
@@ -108,6 +97,8 @@ export const getServerSideProps: GetServerSideProps<NextPageProps> = async (cont
     checkoutSteps,
   }
 
+  console.log('ssp')
```
Comment (robinandeer) [inline apps/store/src/pages/checkout/index.tsx:100]: remove debug comment

File: apps/store/src/services/apollo/client.ts [hunk 4]
```diff
@@ -84,6 +84,7 @@ export const initializeApollo = ({
     ...getDeviceIdHeader({ req, res }),
     ...getAuthHeaders({ req, res }),
   }
+  // console.trace('ah', headers)
```
Comment (robinandeer) [inline apps/store/src/services/apollo/client.ts:87]: remove debug stuff

**Review by robinandeer — APPROVED** [review]
Damn this is moving fast now!

Just clean up the debug logs and we are good ;)

**Comment by alebedev** [comment]
⏳ This pull request is set to merge as part of a Graphite merge job
Stack job ID: LNt7OB37uAsddXEUWHM6.
See details on [graphite.dev](https://app.graphite.dev/github/pr/HedvigInsurance/racoon/1554)

**Pull request merged**